framework/src/timesteppers/IterationAdaptiveDT.C

```diff
@@ -512,24 +517,25 @@ IterationAdaptiveDT::computeAdaptiveDT(Real & dt, bool allowToGrow, bool allowTo
                                       : 0);
   const unsigned int shrink_l_its(_linear_iteration_ratio *
                                   (_optimal_iterations + _iteration_window));
+  const std::string ite_type = (_fe_problem.numLinearSystems() == 0) ? "nl" : "solver";
 
-  if (allowToGrow && (_nl_its < growth_nl_its && _l_its < growth_l_its))
+  if (allowToGrow && (_solver_its < growth_nl_its && _l_its < growth_l_its))
```

Comment:
whether we use `_solver_its` or `_outer_its`, let's rename `growth_nl_its` to match